articles/active-directory/develop/developer-guide-conditional-access-authentication-context.md

```diff
@@ -216,6 +216,47 @@ Do not use auth context where the app itself is going to be a target of Conditio
 - [Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web app](https://github.com/Azure-Samples/ms-identity-dotnetcore-ca-auth-context-app/blob/main/README.md)
 - [Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web API](https://github.com/Azure-Samples/ms-identity-ca-auth-context/blob/main/README.md)
 
+## Authentication Context [ACRs] in Conditional Access expected behavior
+
+### Explicit Auth Context Satisfaction in Requests
+A client can explicitly ask for a token with an Auth Context (ACR) through the claims in the request's body. If an ACR was asked, Conditional Access will allow issuing the token with the asked ACR if all challenges were completed.
+
+### Expected Behavior When an Auth Context Is Not Protected by Conditional Access in The Tenant
+Conditional Access will allow an ACR addition to token's claims if there is no Conditional Access policy enforcing it.
+If there is no Conditional Access policy enforcing an ACR, and no ACR was requested, you can expect that no ACR will be issued.
+ACR requested | Policy applied | Control satisfied | ACR added to claims |
```

Comment:
I like the table. In the empty cells I'd add some text like "none" and "N/A".
I'd also add a row showing cases where there is a policy.
"c2"| Yes | Yes| "c2"
"c3"|Yes|No| Not issued

Reply:
I thought about having a separated section of expected behavior when there is no CA policy protecting an ACR.
Would we prefer combining the explanations into a single table containing both expected when a policy exists and does not?